tests/test_nested_dict.py

```diff
@@ -44,23 +39,8 @@ def test_default(self):
         all = sorted(tup for tup in nd.items_flat())
         self.assertEqual(all, expected_result)
 
-import sys
-if sys.version < '3':
-    import codecs
-
-    def u(x):
```

Comment:
This isnt necessary for the existing tests, and will not be sufficient for extensive tests that ensure nested_dict.__str__ performs correct when the dicts contain unicode